bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -89,22 +89,48 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: CertificateRequest
+    - description: "A CertificateRequest is used to request a signed certificate from\
+        \ one of the configured issuers. \n All fields within the CertificateRequest's\
+        \ `spec` are immutable after creation. A CertificateRequest will either succeed\
+        \ or fail, as denoted by its `status.state` field. \n A CertificateRequest\
+        \ is a one-shot resource, meaning it represents a single point in time request\
+        \ for a certificate and cannot be re-used."
```

Comment:
There seems to be some indentation after the newlines in the descriptions for CertificateRequest and Certificate. 
I considered stripping that, but it renders fine in operatorhub.io and it seems like this should be fixed upstream:

https://raw.githubusercontent.com/cert-manager/cert-manager/master/deploy/crds/crd-certificates.yaml